### ISSUE TITLE
Allow to redefine parameters of the section "Cloud parameters" in the Run... dialog

### ISSUE THREAD

**Issue by schaubl** [slipstream/SlipStreamUI#358]

**Comment by schaubl** [comment]
@slipstream/crew @slipstream/owners 

I am promoting this issue to a 3 stars one because it will improve the SlipStream experience where currently users will get stuck or will have to use annoying workarounds. 

Examples: 
- I want to deploy the "Faces" application from a NuvlaBox on this NuvlaBox but I don't have the same camera as the SixSq one, so VID and PID are different and I cannot specify them when I click on the `Deploy` button. So I have to edit the image. If now I want to plug an other camera, I have to redo the operation.
- I am a newly registred user on Nuv.la and I want to run the WordPress application but with a custom security group which doesn't allow everything, I have to copy the component and then edit it (if it was part of an "application" I should have copied the deployment as well and edited the link to the component). A new SlipStream user will probably not think that it can workaround that way. And also doing it that way we will miss next updates of the WordPress application. BTW now with recipes inheritance we can create a new empty component which inherit from the "Official" one and change the security group field but again I don't think that a new user will have this idea.
- When we deploy a production SlipStream we use an application which will use a big VM but when we want to test it, we may want to use a smaller VM. 

In summary I think fixing this issue will improve user experience of both new and "advanced" users (and it's also useful for us).

This might also require some work server side.
